configs/ote/custom-counting-instance-seg/resnet50_maskrcnn/template.yaml

```diff
@@ -0,0 +1,53 @@
+# Description.
+model_template_id: Custom_Instance_Segmentation_MaskRCNN-ResNet50
+name: MaskRCNN-ResNet50
+task_type: COUNTING
+task_family: VISION
+instantiation: "CLASS"
+summary: MaskRCNN with ResNet50 backbone.
```

Comment:
More descriptive name is required. Please have a look at CVS-76336 for example

Reply:
updated but not sure if it's clear enough